config/Config-template.groovy

```diff
@@ -167,6 +167,20 @@ environments { development { com.recomdata.guestAutoLogin = true } }
 // control the level of access anonymous users will have (the access will be match
 // that of the account specified here).
 com.recomdata.guestUserName = 'guest'
+
+/*
+user.password.strength.regex = '^' +
+//Ensure string has an uppercase letter.
+'(?=.*[A-Z])' +
+//Ensure string has one special case letter.
+'(?=.*[!@#$&*])' + 
+//Ensure string has a digit.
+'(?=.*[0-9])' +
+//Ensure string length is not less then 8.
+'.{8,}' +
+'$'
+*/
```

Comment:
Also don't put commented out directives on `Config-template.groovy`. `Config-template.groovy` only has the default settings not in the in-tree `Config.groovy`. This should be in `Config-extra.php.sample`, which is the file that users copy and modify.